pyvista/jupyter/pv_pythreejs.py

```diff
@@ -265,14 +265,19 @@ def to_surf_mesh(actor, surf, mapper, prop, add_attr={}):
         'wireframe': prop.GetRepresentation() == 1,
         'opacity': prop.GetOpacity(),
         'wireframeLinewidth': prop.GetLineWidth(),
-        # 'side': 'DoubleSide'  # enabling seems to mess with textures
+        # 'side': 'DoubleSide' # enabling seems to mess with textures
```

Comment:
This is a needless (slightly harmful) diff too. [PEP 8](https://www.python.org/dev/peps/pep-0008/#inline-comments) says
>  Inline comments should be separated by at least two spaces from the statement.

So the original was correct, and that's how inline comments appear all across the codebase.

Reply:
Change has been reverted.

Reply:
Yup, thank you. Did you push the revert with another user on purpose, or was that an accident?

Reply:
I don't mind at all, I just wanted to avoid potential mistakes in the commit history for your sake. All good then.

Reply:
👍